### DISCUSSION TITLE
RowSelection Plugin

### DISCUSSION THREAD

**Discussion by ghost** [iamyuu/gridjs-svelte#12] [Q&A]
I would need to be able to select some rows and have the selected items in a list.
I'm able to have the checkbox appear by adding the following columns:

```
{
  id: 'selection',
  name: 'select',
  plugin: {
      component: RowSelection,
      props: {
          id: (row) => row.cell(1).data
      }
}
```

But then I'm struggling to understand how to retrieve the selected items.

Is the existing RowSelection Plugin usable in Svelte?
Thanks!

**Comment by carlos-dubon** [comment]
Just follow the docs, it works.

https://gridjs.io/docs/plugins/selection/row-selection
